precompiles/werc20/werc20.go

```diff
@@ -32,6 +32,13 @@ type Precompile struct {
 	*erc20.Precompile
 }
 
+const (
+	// DepositRequiredGas defines the gas required for the Deposit transaction.
+	DepositRequiredGas uint64 = 28_799
+	// WithdrawRequiredGas defines the gas required for the Withdraw transaction.
+	WithdrawRequiredGas uint64 = 3_000_000
```

Comment:
This is a lot, is that the value for withdrawing?

Reply:
This is what was originally there, I haven't measured the required gas yet. It would be just analyzing the WEVMOS contract gas directly ?

Reply:
Updated it by calling WEVMOS on mainnet a couple of times.